templates/vue/cypress/fixtures/accordion.json

```diff
@@ -215,7 +213,7 @@
       "backgroundColor": "#8396a1",
       "textColor": "white",
       "skippable": true,
-      "quiz": [],
+      
```

Comment:
Please do not add empty line for no reason. This is an issue in a lot of the edited fixtures.